### ISSUE TITLE
Navbar is Not Sticky

### ISSUE THREAD

**Issue by ansiace** [Manishkr1007/WordWeaver#71]
The navbar does not remain fixed at the top of the page when scrolling, which affects the user experience and accessibility. A sticky navbar is essential for easy navigation across different sections of the site.

### Current Behavior
- The navbar scrolls out of view as the user scrolls down the page, requiring users to scroll back up to access navigation links.

### Expected Behavior
- The navbar should remain at the top of the viewport as the user scrolls, providing constant access to navigation options.

![Screenshot 2024-10-15 023210](https://github.com/user-attachments/assets/8f9d2ad0-3901-4b5f-b66c-12b650faa363)

**Comment by JagritiChopra** [comment]
yes you are right .I can fix this .Can you pls assign this to me?

**Comment by ansiace** [comment]
Ok @Manishkr1007  please give me tag of atleast level 2
